templates/topcrash_add_keyword.html

```diff
@@ -1,4 +1,4 @@
-<p>The following {{ plural('bug is', data, pword='bugs are') }} missing a topcrash keyword:
+<p>The following topcrash {{ plural('bug is', data, pword='bugs are') }} missing a topcrash keyword or {{ plural('has', data, pword='have') }} low severity:
```

Comment:
Could we make the message more precise and only mention the actual situation (instead of "or")?

Reply:
Done in f3ec6d67528ad8e85ba4337fe3d3c4d5c0f43e2c